src/formBuilder/utils.js

```diff
@@ -461,8 +461,8 @@ export function generateElementPropsFromSchemas(parameters: {
         definitionData[pathArr[2]]
       ) {
         elementDetails = {
-          ...elementDetails,
           ...definitionData[pathArr[2]],
+          ...elementDetails,
```

Comment:
This ensures that "local" title and definition is favoured over the definition's title and description.